File: 7-kyu/fizz-slash-buzz/fizz-slash-buzz.js
function solution(number){
  let a = []
  let b = []
  let c = []
  for(let i = 1; i < number; i++){
    if(i % 3 === 0 && i % 5 !== 0){
      a.push(i)
    }
    
    if(i % 5 === 0 && i % 3 !== 0){
      b.push(i)
    }
    
    if(i % 5 === 0 && i % 3 === 0){
      c.push(i)
    }
  }
  
  let total = []
  total.push(a.length)
  total.push(b.length)
  total.push(c.length)
  return total
}

File: 7-kyu/fizz-slash-buzz/fizz-slash-buzz.py
def solution(number):
    a = []
    b = []
    c = []
​
    for i in range(1, number):
        if i % 3 == 0 and i % 5 != 0:
            a.append(i)
​
        if i % 5 == 0 and i % 3 != 0:
            b.append(i)
​
        if i % 5 == 0 and i % 3 == 0:
            c.append(i)
​
    total = []
    total.append(len(a))
    total.append(len(b))
    total.append(len(c))
​
    return total